build_tools/benchmarks/post_benchmarks_as_pr_comment.py

```diff
@@ -296,11 +314,19 @@ def check_file_path(path):
       raise ValueError(path)
 
   parser = argparse.ArgumentParser()
-  parser.add_argument("benchmark_files",
+  parser.add_argument("--benchmark_files",
                       metavar="<benchmark-json-file>",
                       type=check_file_path,
+                      default=[],
                       nargs="+",
                       help="Path to the JSON file containing benchmark results")
```

Comment:
There's a weird mix of plural and singular here. Looks like it used to be a single file and is now multiple? Can you clean this up?